app/src/main/res/values/styles.xml

```diff
@@ -23,6 +39,13 @@
         <item name="android:background">#e5e5e5</item>
     </style>
 
+    <style name="SeparatorLineBlack" parent="SeparatorLine">
```

Comment:
you can use `SeparatorLine.Black` then you don't need to write the `parent` attribute.